### DISCUSSION TITLE
when do i get my benefits from the GitHub student developer pack

### DISCUSSION THREAD

**Discussion by Mangoheg** [community/community#106308] [GitHub Education]
### Hi everyone,

i was wondering when I would get my benefits from the student pack; i got accepted on Friday, and i got an email but i don't know if its the right one.

**Comment by ZeeshanMukhtar1** [comment]
Hi @Mangoheg, first of all, congratulations on getting the Student developer pack, U will get the Benefits within the week of selection. 🎊

**Reply by Mangoheg** [reply]
Thank you. In addition, does a teacher have to use GitHub often to become verified?

**Comment by ZeeshanMukhtar1** [comment]
there isn't a specific requirement for teachers to use GitHub often in order to become verified.

**Reply by mishmanners** [reply]
As long as you are a teacher with a .edu account, you should be able to apply and become verified.

**Comment by mishmanners** [comment]
Once you are verified, you should have access soon. You'll receive an email around 72 hours after being accepted, notifying you that your benefits are activated and you should then have access to them.

**Comment by Vikaschoudhary20** [comment]
hello everyone,
My application for student developer pack approved. when will I get benefits from the student pack?
